docs/deploying.md

```diff
@@ -121,6 +121,18 @@ This uses one cache per calendar day (in the `America/Los_Angeles` time zone). I
 
 <div class="note">You’ll need to edit the paths above if you’ve configured a source root other than <code>src</code>.</div>
 
+## Deploying to multiple targets
```

Comment:
This framing feels like it’s assuming a particular use case, but all we’re talking about here is controlling where the `deploy.json` file lives. It was already possible to deploy to “multiple targets” if you didn’t mind overwriting your `deploy.json` file. I think it’s reasonable to describe the staging use case as an example, but I think this section should be called “Deploy configuration” and we should move it up above the **Automated deploys** section and combine it with the existing note (“The deploy command creates a file at .observablehq/deploy.json under the source root…”).

I’ll push some commits to this branch.